web/src/components/PoiDetails.tsx

```diff
@@ -225,10 +226,19 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
           </Collapsible>
         </>
       )}
+      <>
+        <Spacer borderColor={theme.colors.poiBorderColor} />
+        <OpeningHours
+          direction={direction}
+          openingHours={openingHours}
+          isCurrentlyOpened={isCurrentlyOpened}
```

Comment:
I think we should call this `isCurrentlyOpen`. It being opened sounds like that place has had an opening as opposed to it still being in the works.

Reply:
i mean open and opened means `geöffnet` but okay...